src/main/java/fr/ospiea/oscra/employee/object/EnabledType.java

```diff
@@ -0,0 +1,9 @@
+package fr.ospiea.oscra.employee.object;
+
+/**
+ * Created by taozheng on 29/09/2016.
+ */
+public enum EnabledType {
```

Comment:
In fact, I do not sure the type of EnabledType. It may be "activated", "unactivated", "enabled", "disabled".